core/src/main/java/com/amplifyframework/predictions/models/EmotionType.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+/**
+ * Types of emotions that can be detected.
+ */
+@SuppressWarnings("JavadocVariable")
+public enum EmotionType {
```

Comment:
How do these relate to the sentiments? Can we add a text into the doc?

Reply:
emotion and sentiments are different features that are detectable from a given text

Reply:
Please put it in Javadoc, though. I'm not asking for me -- I'm asking for the next person!